lmdeploy/pytorch/kernels/pagedattention.py

```diff
@@ -398,7 +398,7 @@ def paged_attention_fwd(
     q_seqlens: Tensor,
     kv_seqlens: Tensor,
     max_seqlen: int,
-    window_size: int = -1,
+    window_size: int = None,
```

Comment:
Any difference between default -1 and default None?

Reply:
No difference